unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/Common/AssetLibrary_Poolable.cs

```diff
@@ -97,7 +100,13 @@ public override void Release(AssetType asset)
                 return;
             }
 
-            PoolManager.i.Release(asset.container);
+            if ( !poolManager.HasPoolable(asset.container))
+            {
+                Object.Destroy(asset.container);
+                return;
+            }
```

Comment:
We can bring this tidbit to master, as it resolves undeserved PoolManager warnings.